dev-packages/e2e-tests/test-applications/solidstart/package.json

```diff
@@ -3,20 +3,9 @@
   "version": "0.0.0",
   "scripts": {
     "clean": "pnpx rimraf node_modules pnpm-lock.yaml .vinxi .output",
-    "clean:build": "pnpx rimraf .vinxi .output",
     "dev": "NODE_OPTIONS='--import ./src/instrument.server.mjs' vinxi dev",
-    "build": "vinxi build",
-    "//": [
-      "We are using `vinxi dev` to start the server because `vinxi start` is experimental and ",
-      "doesn't correctly resolve modules for @sentry/solidstart/solidrouter.",
-      "This is currently not an issue outside of our repo. See: https://github.com/nksaraf/vinxi/issues/177",
-      "We run the build command to ensure building succeeds. However, keeping",
-      "build output around slows down the vite dev server when using `@sentry/vite-plugin` so we clear it out",
-      "before actually running the tests.",
-      "Cleaning the build output should be removed once we can use `vinxi start`."
-    ],
-    "preview": "pnpm clean:build && HOST=localhost PORT=3030 NODE_OPTIONS='--import ./src/instrument.server.mjs' vinxi dev",
-    "start": "HOST=localhost PORT=3030 NODE_OPTIONS='--import ./src/instrument.server.mjs' vinxi start",
```

Comment:
we don't need the start command anymore?

Reply:
I see that the playwright config has `pnpm preview` as a start command, so this is probably not necessary.

Reply:
Yea, what @s1gr1d said. I had the start in there just for local testing but it's not necessary anymore.